guru.js

```diff
@@ -10,7 +10,7 @@ module.exports = () => {
 
     guru.use(express.json());
 
-    guru.use(session({ secret: 'cats' }));
+    guru.use(session({ secret: 'guest' }));
```

Comment:
As funny as this is.... please reference an environment variable instead

Reply:
We currently commit our .env, there is a task to remove that, and start using our .env correctly so we can store these types of values in it securely.

Reply:
I can reference a variable to better prepare for those changes, which I'll do, but it won't be any more secure until then.